### DISCUSSION TITLE
Hero 10 Timestamp overlay

### DISCUSSION THREAD

**Discussion by cam867** [gopro/labs#883] [General]
Overlay burn-in is not working on HERO10?

Hello,

​I have a HERO10 I'm trying to add the timestamp on recording. I've installed Lab firmware then using the overlay(time & date) QR command but it seems not working, below are what I've tried:

​
    Latest version of HERO10 and Lab firmware
    Made sure the video format is meeting the requirement of overlay(4K 30、2.7K 60)
    Also tried 1080/30

 

When I scan the QR code, screen will show "Overlay Active" but the overlay is not on the recorded video. I recorded a 1 minute segment. 

 
QR Command:
https://gopro.github.io/labs/control/set/?cmd=g0!MBRNO=10!MBURN="(0,20)[BRHH:MM:SS yyyy-mm-dd]"&title=Overlays

​Is there anything I can do?

**Comment by dnewman-gpsw** [comment]
It works in my testing
![image](https://github.com/gopro/labs/assets/10856084/a39fbbe0-2388-4780-ad31-0ef59b065042)

**Comment by cam867** [comment]
Are there any logs that I can review to see why it's not working on my camera?

**Reply by dnewman-gpsw** [reply]
What firmware version are you running?

**Comment by cam867** [comment]
Version: 01.50.70

The timestamp did not show up when playing the videos back in the GoPro App on my phone but I know see them when I download the videos to my computer. Thanks.

**Reply by dnewman-gpsw** [reply]
Correct. They are only rendered in the full resolution video, not the preview that Quik uses.

**Comment by DavidFolan** [comment]
Is it possible to increase time resolution to include milliseconds (for sync purposes)?  I've tried adding ss and sss after the timecode but that doesn't work

**Reply by dnewman-gpsw** [reply]
It is not possible, the screen update for overlay is not always frame accurate, so add milliseconds would only make it less precise.

**Reply by Martin22-ye** [reply]
Is there any option to know the miliseconds or frame accuracy?

**Reply by dnewman-gpsw** [reply]
For what application? The camera with GPS, have UTC time at 10Hz, so you can determine frame time to about 100ms precision.

**Reply by Martin22-ye** [reply]
thanks for answering. It turns out that I need to get the start, finish and 4 split times of a race with several runners. I want to place a go pro hero 12 at the start, each split and at the finish line. From the analysis of the video I could extract the times (as accurate as possible) of all the splits and the finish from the athletes.

How would you do it to have the maximum precision between the cameras if I am recording with 1080p and 60fps and hero pro 12?

**Reply by dnewman-gpsw** [reply]
No GPS in HERO12, but you don't absolute time reference (UTC) just camera to camera.  Can you just sync all the camera from the start gun in post?

**Reply by Martin22-ye** [reply]
Okay, so by synchronizing the cameras beforehand using Precision Time (https://gopro.github.io/labs/control/precisiontime/), recording the video with the different cameras in diferent zones, and then editing all the videos in the timeline during post-production, it should work for capturing all times. 

I am right?

**Reply by dnewman-gpsw** [reply]
Yes. This is want the Precision time help enable.  You can use timecode embedded in the MP4 to have a sync'd edit.  Only warning, the timecode precision (use precision time sync) is ±1 frame accurate, if the camera are not powered off between the scan and capture. If the cameras power down, the clock time is only ±1 second accurate.

**Reply by Martin22-ye** [reply]
What do yo mean with "You can use timecode embedded in the MP4 to have a sync'd edit"?

**Reply by dnewman-gpsw** [reply]
95% of good video editors extract timecode from the MP4 automatically. It's how these videos are synced, I presume this is the reason you are synchronizing the time.
